custom_components/adjustable_bed/beds/nectar.py

```
@@ -0,0 +1,330 @@
+            try:
+                await self.client.write_gatt_char(
+                    NECTAR_WRITE_CHAR_UUID, command, response=True
+                )
+            except BleakError:
+                _LOGGER.exception("Failed to write command")
+                raise
+
+            if i < repeat_count - 1:
+                await asyncio.sleep(repeat_delay_ms / 1000)
+
+    async def start_notify(self, callback: Callable[[str, float], None]) -> None:
```

Ruff (0.14.11)

Unused method argument: `callback`

(ARG002)